dokka-integration-tests/utilities/build.gradle.kts

```diff
@@ -16,6 +16,8 @@ dependencies {
     implementation(libs.kotlinx.coroutines.core)
     implementation(libs.jsoup)
     implementation(libs.eclipse.jgit)
+
+    implementation(gradleTestKit())
```

Comment:
why it's needed here? `utilities` is shared between other runners integration tests?

Reply:
it was required for gradleRunnerUtils.kt, but that's now moved to the gradle integration test project